src/main/java/org/jenkinsci/extension_indexer/SourceAndLibs.java

```diff
@@ -143,6 +143,8 @@ private static void downloadDependencies(File pomDir, File destDir) throws IOExc
             process = System.getenv("M2_HOME") + "/bin/mvn";
         }
         ProcessBuilder builder = new ProcessBuilder(process,
+                "--settings", new File("maven-settings.xml").getAbsolutePath(),
+                "--update-snapshots",
                 "dependency:copy-dependencies",
```

Comment:
Maybe try running the `validate` phase first:

    mvn -s … -U -DincludeScope=compile -DoutputDirectory=… validate dependency:copy-dependencies